client/internal/engine.go

```diff
@@ -388,15 +388,15 @@ func signalCandidate(candidate ice.Candidate, myKey wgtypes.Key, remoteKey wgtyp
 	return nil
 }
 
-func signalAuth(uFrag string, pwd string, myKey wgtypes.Key, remoteKey wgtypes.Key, s signal.Client, isAnswer bool) error {
+func signalAuth(uFrag string, pwd string, myKey wgtypes.Key, myPort int, remoteKey wgtypes.Key, s signal.Client, isAnswer bool) error {
```

Comment:
I think that we should be more explicit in calling this field. 
E.g. WireGuard listen port or native port